www/js/services.js

```diff
@@ -215,78 +117,4 @@ angular.module('emission.services', ['emission.plugin.logger'])
       return window.cordova.plugins.BEMConnectionSettings.getSettings();
     };
 
-})
-
-.factory('Chats', function() {
-  // Might use a resource here that returns a JSON array
-
-  // Some fake testing data
```

Comment:
Thanks for doing this! FYI, this was part of the original ionic starter project that I used to create the the very first version of the app. I kept it around for a bit as an example, and then nobody ever had the time/chutzpah to clean it up 😄 

I'm glad we are finally getting around to it!